pages/establishment/dashboard/views/index.jsx

```diff
@@ -49,20 +51,44 @@ const Index = ({
           <PanelList
             panels={links.filter(link => allowedActions.includes(link.permissions)).map((link, index) => <DashboardLink key={index} { ...link } />)}
           />
+          {
+            canApply &&
+              <Link page="establishment.apply" label={<Snippet>buttons.establishment.apply</Snippet>} className="govuk-button" />
+          }
+          {
+            allowedActions.includes('establishment.update') && openApplication &&
```

Comment:
```suggestion
            openApplication &&
```

Reply:
Since the other rendering condition rolls in a permissions check it makes sense to do the same for this one.